docs/selectively-triggering-aktualizr.adoc

```diff
@@ -1,20 +1,20 @@
 = Selectively triggering aktualizr
 
-Aktualizr normally runs as a systemd service and regularly polls for updates. The default polling interval is set at 10s, and this should be increased by a few orders of magnitude in a production system.
+Aktualizr normally runs as a `systemd` service and regularly polls for updates. The default polling interval is set at 10s, and this should be increased by a few orders of magnitude in a production system.
 
 It is also possible to trigger the update cycle, or individual parts of the update cycle, manually or programmatically. This is done by using aktualizr's `--running-mode` option (or by setting `running_mode` in the `[uptane]` configuration section).
 
 [NOTE]
 ====
-To selectively trigger aktualizr, you should disable the systemd service; a continuously-running aktualizr will interfere with the manual triggering.
+To selectively trigger aktualizr, you should disable the `systemd` service. If aktualizr runs continuously, it interferes with the manual triggering.
 
 To run the update cycle manually, first stop and disable the aktualizr background service:
 
     systemctl stop aktualizr && systemctl disable aktualizr
 
-Assuming aktualizr is being built using Yocto and meta-updater, the systemd service can also be disabled by default by adding the following to `local.conf`:
+Assuming aktualizr is being built using Yocto and meta-updater, the `systemd` service can also be disabled by default by adding the following to `local.conf`:
 
-    SYSTEMD_AUTO_ENABLE_aktualizr = "disable"
+    `systemd`_AUTO_ENABLE_aktualizr = "disable"
```

Comment:
The backticks are probably undesirable in this instance, unless you want to wrap all of the similar commands and configuration bits completely in backticks.

Reply:
ah sorry, that was a find and replace mistake. it shouldn't have been there.

Reply:
fixed.